src/information_resource_registry/schema/information_resource_registry.yaml

```diff
@@ -76,6 +78,22 @@ classes:
 
 slots:
 
+  consumes:
```

Comment:
@andrewsu - good definitions for these new slots?

Reply:
Looks good to me!